include/cantera/numerics/Integrator.h

```diff
@@ -288,23 +288,34 @@ class Integrator
         return stats;
     }
 
+    virtual void setMaxNonlinIterations(int n) {
+        warn("setMaxNonlinIterations");
```

Comment:
This is consistent with how other methods that aren't implemented in the `Integrator` base class are handled -- the `warn` method is a private member of `Integrator` used only for this purpose. It might make more sense for all of these to throw `NotImplementedError`, but I don't know that this PR is the time to take care of that cleanup.